wiki/Announcements/Contests/Monthly_Fanart_Contest/A_very_osu!_halloween/en.md

```diff
@@ -0,0 +1,95 @@
+# A very osu! Halloween
```

Comment:
regarding the title, the behavior for titles appears to be this:
`parse the markdown` -> `remove level 1 header` _(idk, if it is only the first one or all of them)_ (this also explains why it isn't part of the content body, rather that top banner) -> `use the folder path as the main folder, if any` (the smaller heading) -> `use the path after that folder then end at the locale.md file, if any` (the bigger heading) ... if there is no subfolder, use small heading as the bigger heading

if anything really, just type in anything in the url to see what it will look like (the page will 404 but osu-web will parse the path anyways for the title):

- http://new.ppy.sh/wiki/Announcements/Contests/Monthly_Fanart_Contest/A_very_osu!_halloween/
- http://new.ppy.sh/wiki/Welcome/to/osu!/this/page/does/not/exist/but/hey/this/is/an/example/to/see/what/happens/to/articles/inside/two/or/more/subfolders/_(%E2%95%AF%C2%B0%E2%96%A1%C2%B0%EF%BC%89%E2%95%AF%EF%B8%B5_%E2%94%BB%E2%94%81%E2%94%BB

this approach may be unwanted since translated titles will have no purpose... but the current is to simulate the pathing